src/routes/search.js

```diff
@@ -83,12 +97,13 @@ class SearchRoute extends Component {
   getResults() {
     let { resolver } = this.props;
     let { searchType, params } = this.state;
-    let { offset = 0, ...queryParams } = params;
+    let searchParams = this.transformQueryParams(params);
+    let { offset = 0, ...pagedParams } = searchParams;
     let page = Math.floor(offset / 25) + 1;
 
     return resolver.query(searchType, {
       filter: undefined,
-      ...queryParams,
+      ...pagedParams,
```

Comment:
This can be `searchParams` once we change that transform. `pagedParams` is sort of a misnomer since it doesn't actually include any page parameter.